### ISSUE TITLE
Docker image `archlinux/base` no longer exists.

### ISSUE THREAD

**Issue by soufianefariss** [scrt/avcleaner#13]
Faced this error while compiling the project:
```
$ docker build . -t avcleaner
Sending build context to Docker daemon  108.3MB
Step 1/7 : FROM archlinux/base
pull access denied for archlinux/base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

I think the Docker image `archlinux/base` no longer exists.

**Comment by asceylan** [comment]
You can edit the Dockerfile and remove "/base" from the sentence . 
You Docker Image name in Dockerfile should be like :
  
FROM archlinux

**Comment by soufianefariss** [comment]
I figured it has be renamed to `archlinux`, though building `avcleaner` still fails.

```
$ make -j2
[ 33%] Building CXX object CMakeFiles/avcleaner.bin.dir/main.cpp.o
[ 33%] Building CXX object CMakeFiles/avcleaner.bin.dir/MatchHandler.cpp.o
/home/toto/main.cpp: In member function ‘virtual void AVObfuscator::Action::EndSourceFileAction()’:
/home/toto/main.cpp:136:85: error: conversion from ‘llvm::StringRef’ to non-scalar type ‘std::string’ {aka ‘std::__cxx11::basic_string<char>’} requested
  136 |             std::string FileName = SM.getFileEntryForID(SM.getMainFileID())->getName();
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
/home/toto/MatchHandler.cpp: In static member function ‘static std::vector<std::__cxx11::basic_string<char> > MatchHandler::getNodeParents(const clang::StringLiteral&, clang::ast_type_traits::DynTypedNode, clang::ASTContext*, std::vector<std::__cxx11::basic_string<char> >&, uint64_t)’:
/home/toto/MatchHandler.cpp:33:17: error: ‘DynTypedNodeList’ is not a member of ‘clang::ASTContext’
   33 |     ASTContext::DynTypedNodeList parents = Context->getParents(NodeString);
      |                 ^~~~~~~~~~~~~~~~
/home/toto/MatchHandler.cpp:36:9: error: ‘parents’ was not declared in this scope
   36 |         parents = Context->getParents(Node);
      |         ^~~~~~~
/home/toto/MatchHandler.cpp:39:31: error: ‘parents’ was not declared in this scope; did you mean ‘parent’?
   39 |     for (const auto &parent : parents) {
      |                               ^~~~~~~
      |                               parent
/home/toto/MatchHandler.cpp:48:48: error: no matching function for call to ‘std::vector<std::__cxx11::basic_string<char> >::push_back(llvm::StringRef& ’
   48 |         CurrentParents.push_back(ParentNodeKind);
      |                                                ^
```

**Comment by plowsec** [comment]
Hey, the last bug should be fixed when #10 is fixed. For the first one, I'll update the Dockerfile right away.
